### PR TITLE
Update snakeGame.js

### PR DESCRIPTION
**Pull request by travis-horton**
i deleted my whole game so everyone can see the whole code...

### CODE REVIEW

File: snakeGame.js [hunk 1]
```diff
@@ -1,101 +1 @@
...
-    appleLoc: vecInBoard(),
-    snakeLoc: [[boardSize/2, boardSize/2]],
-    snakeMove: "right"
-}
-
-window.addEventListener("keydown", recordInput);
-
-function recordInput(e) {
-    switch (e) {
-        case 37:
```
Comment (jmou) [inline snakeGame.js:23]: Are there key constants?

File: snakeGame.js [hunk 2]
```diff
@@ -1,101 +1 @@
...
-    console.log(appleLoc);
-
-    ctx.style = "rgb(255, 0, 0)";
-    ctx.rect(appleLoc[0], appleLoc[1], 10, 10);
-    ctx.fill();
-}
-
-function snakeNextLoc(currLoc, dir) {
-    switch(dir) {
-        case "left":
```
Comment (jmou) [inline snakeGame.js:78]: Does javascript have enums?

File: snakeGame.js [hunk 3]
```diff
@@ -1,101 +1 @@
...
-            return currLoc[1] - 1;
-        case "right":
-            return currLoc[0] + 1;
-        case "down":
-            return currLoc[1] + 1;
-
-    }
-}
-
-function vecInBoard() {
```
Comment (jmou) [inline snakeGame.js:90]: maybe indicate this is a random location, like "randomCoord"?

File: snakeGame.js [hunk 4]
```diff
@@ -1,101 +1 @@
...
-        case 38:
-            state.snakeMove = "up";
-        case 39:
-            state.snakeMove = "right";
-        case 40:
-            state.snakeMove ="down";
-    }
-}
-
-tick();
```
Comment (jmou) [inline snakeGame.js:34]: is there a loop or timer for running ticks?

File: snakeGame.js [hunk 5]
```diff
@@ -1,101 +1 @@
...
-body.appendChild(c);
-let boardSize = 20;
-c.id = "c";
-c.width = boardSize * 10;
-c.height = boardSize * 10;
-c.style.border = "1px solid black";
-var ctx = c.getContext("2d");
 
-let state = {
-    snakeLength: 2,
```
Comment (jmou) [inline snakeGame.js:13]: is this used?

File: snakeGame.js (hunk 2 above)
Comment (travis-horton) [reply snakeGame.js:78]: yes... i will change that. :)

File: snakeGame.js (hunk 3 above)
Comment (travis-horton) [reply snakeGame.js:90]: that's a better name! :)

File: snakeGame.js (hunk 4 above)
Comment (travis-horton) [reply snakeGame.js:34]: yes there is a loop for running ticks, but i didn't get there yet! :)

**Pull request closed without merging**